geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/docker-compose.yaml

```diff
@@ -109,6 +109,8 @@ services:
 
   # Rich image for project development with e.-g. vim, tree, awscli, psql, ...
   tools:
+    volumes:
+      - .:/src
```

Comment:
This volume is already present in `docker-compose-lib.yaml`
Do I miss something ?

Reply:
Arg :-/